instrumentation/spring/spring-rmi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springrmi/server/ServerInstrumentationModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.springrmi.server;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ServerInstrumentationModule extends InstrumentationModule {
+
+  public ServerInstrumentationModule() {
+    super("spring-rmi", "spring-rmi-server");
```

Comment:
the rmi instrumentation itself is slightly different from normal convention, here's a good example similar to this one: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/39d20b5fbc79ab38b7ce685fd5f2610e961ba1e6/instrumentation/spring/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/client/WebfluxClientInstrumentationModule.java#L19

```suggestion
    super("spring-rmi", "spring-rmi-5.0", "spring-rmi-server");
```